modules/tiles/src/tileset/traversers/tileset-traverser.js

```diff
@@ -307,28 +284,36 @@ export default class TilesetTraverser {
     let allDescendantsLoaded = true;
     const stack = this._emptyTraversalStack;
 
+    stack.push(root);
+
     while (stack.length > 0) {
       const tile = stack.pop();
 
+      /* TODO: This doesn't seem needed, should check further.
```

Comment:
Would be great if we can sort this out.

Reply:
So far, at least in my test cases, it seems to work well. The tiles are updated in the main traversal and I don't see any glitches.

Reply:
These turned out to be needed after-all in a more complex model.